app/main.py

```diff
@@ -1,3 +1,82 @@
-def shop_trip():
-    # write your code here
-    pass
+from app.car import Car
+from app.customer import Customer
+from app.shop import Shop
+from app import parsing
+
+import datetime
```

Comment:
Order your imports correctly according to PEP-8.

Reply:
not fixed